### PR TITLE
Added a change to support Yaesu GS232 commands on Elevation

### PR DESCRIPTION
**Pull request by ea4tx**
Hi Tobi,

I have done some changes in order to support the Yaesu GS232 (elveation command)

Pablo EA4TX

### CODE REVIEW

File: rotator/yaesu/yaesu.go [hunk 1]
```diff
@@ -381,8 +381,9 @@ func (r *Yaesu) SetElevation(el int) error {
 
 	r.elPreset = el
 
-	if _, err := r.write([]byte(fmt.Sprintf("N%.3d\r\n",
-		r.elPreset))); err != nil {
+	//if _, err := r.write([]byte(fmt.Sprintf("N%.3d\r\n",
```
Comment (dh1tw) [inline rotator/yaesu/yaesu.go]: line can be removed

File: .remoteRotator.toml [hunk 2]
```diff
@@ -12,19 +12,19 @@ port = 3333
 [http]
 enabled = true
 host = "127.0.0.1"
-port = 7070
+port = 7071
```
Comment (dh1tw) [inline .remoteRotator.toml]: I would prefer to not modify the default values in the `.remoteRotator.toml` config file

Comment (ea4tx) [reply .remoteRotator.toml]: Hi Tobi,

Line 384 removed! 
And Ok about the configuration file. 

Rgds, Pablo EA4TX

**Comment by dh1tw** [comment]
Pablo, you can implement the changes in the same branch and just push them in another commit. No need to create another PR. Just commit them into your remote (Github) repo and the additional commit will show up in this Pull Request.

**Pull request closed without merging**